libc/semantics/settings.k

```diff
@@ -40,13 +39,15 @@ module C-SETTINGS
      rule cfg:alignofFloat => 4
      rule cfg:alignofDouble => 8
      rule cfg:alignofLongDouble => 16
-     rule cfg:alignofEnum => cfg:alignofInt
      rule cfg:alignofPointer => 4
      // Max of the above.
      rule cfg:alignofMalloc => 16
 
-     rule cfg:intToPointer(_, _) => trap
+     rule cfg:intToPointer(0, _) => NullPointer
+     rule cfg:intToPointer(_, _) => trap [owise]
```

Comment:
does this mean you cannot cast a pointer into an integer and back into the same pointer?